src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class Application {
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball = new Ball();
+        System.out.println(ball.getRandomBall().toString());
+        System.out.println(ball.getRandomBall().toString());
+        System.out.println(ball.getRandomBall().toString());
```

Comment:
```suggestion
        System.out.println(ball.getRandomBall().toString());
        System.out.println(ball.getRandomBall().toString());
        System.out.println(ball.getRandomBall().toString());
```
Use loop for this

You don't need to call toString() when you pass an object to the println() method, it gets called automatically